### PR TITLE
fix: add back env variable required for prod build

### PR DESCRIPTION
**Pull request by aronhoyer**

### CODE REVIEW

**Review by wenche — APPROVED** [review]
LGTM 👍

**Review by nilsml — COMMENTED** [review]
Just a question for now

File: Dockerfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ RUN npm ci
 
 FROM node:lts-alpine AS builder
 WORKDIR /opt/app
+ENV NEXT_PUBLIC_COLLIBRA_BASE_URL https://equinor-dev.collibra.com/rest/2.0
```
[the remark this answers is not in the document]
Comment (aronhoyer) [reply Dockerfile:9]: yes, but this was the quickest way and fewest lines to fix prod

[the remark this answers is not in the document]
Comment (nilsml) [reply Dockerfile:9]: I guess to make it work now it is ok, but we should use NEXT\_PUBLIC only when needed

**Pull request merged**